config/initializers/rack_attack.rb

```diff
@@ -8,9 +8,14 @@ class Attack
       IPAddr.new(x)
     end.freeze
 
-    EMAIL_REGISTRATION_PATHS = ['/sign_up/enter_email', '/en/sign_up/enter_email',
-                                '/es/sign_up/enter_email', '/fr/sign_up/enter_email'].freeze
-    SIGN_IN_PATHS = ['/', '/en', '/es', '/fr'].freeze
+    EMAIL_REGISTRATION_PATHS =
+      IdentityConfig.store.available_locales.reduce(['/sign_up/enter_email']) do |paths, locale|
```

Comment:
I guess it's moot since we're already referencing `IdentityConfig` elsewhere in this file, but is there any guarantees that the configuration will be loaded by the time this file is evaluated?

Reply:
Yeah, that's a good point. I added a spec in 5f430ba7ea to try to get at that a little bit?

Reply:
That works 👍